### PR TITLE
Add code sample for landing page

### PR DESCRIPTION
**Pull request by bidoubiwa**
fixes: #9

## But should we?

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  const searchClient = instantMeiliSearch(
+    "http://localhost:7700",
+    "searchKey"
+  )
+
+  @Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css'],
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:15]: Can we omit this style thing?

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  })
+  export class AppComponent {
+    title = 'angular-app'
+    config = {
+      indexName: 'movies',
+      searchClient,
+    }
+  }
+
+  // In app.component.ts
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:25]: The same as the first comment?  `// In app.component.ts` so this entire example is just one file?

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+landing_getting_started_1: |-
+  // In app.component.ts
+
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:3]: I think you could remove this blank line :)

File: .code-samples.meilisearch.yaml (hunk 1 above)
Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml:15]: See algolia: https://www.algolia.com/doc/guides/building-search-ui/getting-started/angular/

**Review by mdubus — APPROVED** [review]

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**